### ISSUE TITLE
Configuration for OVH DNS Challenge

### ISSUE THREAD

**Issue by tadikwa** [tobychui/zoraxy#161]
I have some problems to configure OVH DNS Challenge

I have this field with the ACME TOOL : FieldType, SubDomain, Target and Zone
![2024-05-20_15h00_57](https://github.com/tobychui/zoraxy/assets/6054289/333d8666-1a00-425f-9bea-325a70dc5d2e)

But, normally with Nginx Proxy Manager (for example), i need to use this variables : application_key, application_secret, consumer_key and endpoint
![2024-05-20_15h01_25](https://github.com/tobychui/zoraxy/assets/6054289/5ff629b9-70d8-42fa-953b-b085bf9014c2)
![2024-05-20_15h09_09](https://github.com/tobychui/zoraxy/assets/6054289/48201bb2-fb5f-4c70-8ed8-1603c0cbb30c)

It is a bug of config parser or i didn't understand how to configure ?

**Comment by tobychui** [comment]
This seems like a bug in the config parser. Let me check it in a moment.

**Comment by tadikwa** [comment]
Ah yes ok I see.
However, I am running the version in Docker.
Can I also make the modification on the online container?

**Comment by tobychui** [comment]
@tadikwa I guess you can. I am not experience with docker but in theory this should work.

1. Cross compile Zoraxy on your computer for your server's CPU architecture & OS, e.g. `GOOS=linux GOARCH=amd64 go build -o zoraxy-linux-amd64`
2. ssh into your container, stop zoraxy
3. Transfer the new binary from your computer to the container and overwrite the old version of zoraxy binary executable
4. Restart zoraxy service

**Comment by tadikwa** [comment]
ok, this is out of my scope.

I prefer to wait for a new build and update my docker image ;)

**Comment by tobychui** [comment]
@tadikwa As this is only a front-end issue, you can hack it by using the F12 developer console and enter this.

```javascript
$.ajax({
        url: "/api/acme/autoRenew/setDNS",
        method: "POST",
        data: {
          filename: your_domain_with_*_replaced_by_underscore,
          dnsProvider: "ovh",
          dnsCredentials: JSON.stringify({
               "ApplicationKey":"",
               "ApplicationSecret":"",
               "ConsumerKey":""
          }),
        }
});
```

**Comment by PassiveLemon** [comment]
You could clone the repo, change the line, and then rebuild the container

**Comment by barto95100** [comment]
Maybe upgrade the official container zoraxy to have this change ? most simply for all user :)

**Comment by barto95100** [comment]
any news ?

**Comment by tobychui** [comment]
> any news ? 

FYI, the changes required for fixing this bug has already been pushed into the main branch. 

Release build are not gonna be that fast though (still working on something related to other issues)

**Comment by barto95100** [comment]
OK I understand

it's a shame since I opened the first ticket to indicate the wildcard problem with ovh and when the feature is available there is a bug (we are cursed lol)

we'll wait... (ovh users)

**Comment by tobychui** [comment]
Fixed in v3.0.5.
